submissions/lalalilulu/memory-game/styles.css

```diff
@@ -0,0 +1,146 @@
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@500;700&display=swap');
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body {
+    height: 90vh;
+    background-color: linen;
+    font-family: Manrope;
+    color: #314455;
+}
+
+.title{
+    display: flex;
+    width: 100%;
+    font-size: 3rem;
+    font-weight: 700;
+    text-transform: uppercase;
+    justify-content: center;
+    margin: 2rem 0;
+}
+
+.license {
+    display: flex;
+    width: 100%;
+    font-size: 1rem;
+    font-weight: 500;
+    justify-content: center;
+    margin-top: 2rem;
+    white-space: pre;
+}
+
+.license a {
+    color: #314455;
+}
+
+img {
+    width: 4rem;
+}
+
+.memory-game {
+    width: 50%;
+    height: 75%;
+    display: flex;
+    flex-wrap: wrap;
+    margin: auto;
+    perspective: 1000px;
+}
+
+.memory-card {
```

Comment:
add `cursor: pointer;` for the hover state, please

Reply:
done